tests/detect-flow-pkts/README.md

```diff
@@ -0,0 +1,10 @@
+Test
+====
+
+Test `flow.pkts_toclient` keyword
```

Comment:
I think it's good to list the other words that are being tested :P

Reply:
Just force-pushed a new version, is it ok ?

Reply:
Works for me :P